### DISCUSSION TITLE
Show Drawer programaticallty like a Dialog

### DISCUSSION THREAD

**Discussion by diegoaraujolima** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2707] [Q&A] [answered]
Hello. Actually is possible show a dialog programatically though DialogHost.Show(). How can I do the same with a DrawerHost?

**Comment by Keboo** [comment, marked as answer]
There is not a static method for it. Topically you would simply set the appropriate *IsOpen property to open and close it. This could either be done with data binding (binding to a boolean property on your view model) or simply accessing the drawer host directly in your code behind.
